disdrodb/L0/readers/FRANCE/parser_ACTRIS-CCRES.py

```diff
@@ -0,0 +1,316 @@
+        The default is False.
+    lazy : bool
+        Whether to perform processing lazily with dask.
+        If lazy=True, it employed dask.array and dask.dataframe.
+        If lazy=False, it employed pandas.DataFrame and numpy.array.
+        The default is True.
+
+    Additional information:
+    - The campaign name must semantically match between:
+       - The ends of raw_dir and processed_dir paths
+       - The attribute 'campaign' within the metadata JSON file.
+    - The campaign name are set to be UPPER CASE.
+
+    """
+    ####----------------------------------------------------------------------.
+    ###########################
+    #### CUSTOMIZABLE CODE ####
+    ###########################
+    #### - Define raw data headers
+    # Notes
+    # - In all files, the datalogger voltage hasn't the delimeter,
+    #   so need to be split to obtain datalogger_voltage and rainfall_rate_32bit
+    column_names = ["TO_SPLIT"]
+
+    ##------------------------------------------------------------------------.
+    #### - Define reader options
+    reader_kwargs = {}
+    # - Define delimiter
+    reader_kwargs["delimiter"] = "\\n"
+
+    # - Avoid first column to become df index !!!
+    reader_kwargs["index_col"] = False
+
+    # - Define behaviour when encountering bad lines
+    reader_kwargs["on_bad_lines"] = "skip"
+
+    # - Define parser engine
+    #   - C engine is faster
+    #   - Python engine is more feature-complete
+    reader_kwargs["engine"] = "python"
+
+    # - Define on-the-fly decompression of on-disk data
+    #   - Available: gzip, bz2, zip
+    reader_kwargs["compression"] = "infer"
+
+    # - Strings to recognize as NA/NaN and replace with standard NA flags
+    #   - Already included: ‘#N/A’, ‘#N/A N/A’, ‘#NA’, ‘-1.#IND’, ‘-1.#QNAN’,
+    #                       ‘-NaN’, ‘-nan’, ‘1.#IND’, ‘1.#QNAN’, ‘<NA>’, ‘N/A’,
+    #                       ‘NA’, ‘NULL’, ‘NaN’, ‘n/a’, ‘nan’, ‘null’
+    reader_kwargs["na_values"] = ["na", "", "error"]
+
+    # - Define max size of dask dataframe chunks (if lazy=True)
+    #   - If None: use a single block for each file
+    #   - Otherwise: "<max_file_size>MB" by which to cut up larger files
+    reader_kwargs["blocksize"] = None  # "50MB"
+
+    # Skip first row as columns names
+    reader_kwargs["header"] = None
+
+    # Define encoding
+    reader_kwargs["encoding"] = "latin1"
+
+    ##------------------------------------------------------------------------.
+    #### - Define facultative dataframe sanitizer function for L0 processing
+    # - Enable to deal with bad raw data files
+    # - Enable to standardize raw data files to L0 standards  (i.e. time to datetime)
+    df_sanitizer_fun = None
+
+    def df_sanitizer_fun(df, lazy=False):
+        # Import dask or pandas
+        if lazy:
+            import dask.dataframe as dd
+        else:
+            import pandas as dd
+
+        # Remove header columns
+        df = df[
```

Comment:
The header appear just at the beginning of the file right? 
So I think the header column can be removed through some `reader_kwargs ` light `skiprows `. Please correct for this